src/database/migrations/20190704115631-create-users.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
Nice work, Is there a reason why you are running this new migrations on the `users` table? I do not see a difference between this and the current one on the repository.